darshan-util/pydarshan/.codecov.yml

```diff
@@ -0,0 +1,22 @@
+# Codecov configuration to make it a bit less noisy
+codecov:
+  # don't fail the CI
+  require_ci_to_pass: yes
+
+# default coverage settings
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
+
+# default parser settings
+parsers:
+  gcov:
```

Comment:
`gcov` is for line coverage of compiled code, but you haven't activated coverage builds of the C source underneath the hood, so I'm not sure I understand the motivation for the Python side of things.

We could discuss that if we want it for the C code eventually, but that's probably another can of worms we don't want to deal with initially.

Also, did you check that `.codecov.yml` is properly picked up when not at the root of the repository proper (it is usually placed there)